doc/source/whatsnew/v1.1.0.rst

```diff
@@ -340,6 +340,7 @@ Other enhancements
 - :class:`pandas.core.window.ExponentialMovingWindow` now supports a ``times`` argument that allows ``mean`` to be calculated with observations spaced by the timestamps in ``times`` (:issue:`34839`)
 - :meth:`DataFrame.agg` and :meth:`Series.agg` now accept named aggregation for renaming the output columns/indexes. (:issue:`26513`)
 - ``compute.use_numba`` now exists as a configuration option that utilizes the numba engine when available (:issue:`33966`)
+- :meth:`DataFrame.agg` and :meth:`Series.agg` now accept named aggregation for renaming the output columns/indexes. (:issue:`26513`)
```

Comment:
is this a leftover?

Reply:
@jreback I probably messed something up during merge conflict handling. Reverted this line to the documentation update I made in https://github.com/pandas-dev/pandas/pull/34514/commits/fa4915b7e39251b6ad6d19cdd6355009f94bce80